src/logger.rs

```diff
@@ -192,7 +204,6 @@ impl log::Log for Logger {
     }
 }
 
-// TODO(tisonkun): logback and log4j2 support custom error handling (status listener).
```

Comment:
@andylokandy I finally tend not to add something like ErrorListener or StatusListern for customizing error handling.

The current hard code behavior works well and I don't find any other way to "customize" it. If there is some real requirement (and it's always doable), we can add it later, instead of providing a public interface at this point.